scripts/scylla-get.sh

```diff
@@ -0,0 +1,193 @@
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+set -euo pipefail
+
+main() {
+  require_cmd curl
+
+  SCYLLA_VERSION_RAW=$(curl -s https://repositories.scylladb.com/scylla/check_version?system=scylla)
+
+  SCYLLA_VERSION=$(echo $SCYLLA_VERSION_RAW | sed -e "s/.*version\":\"\(.*\)\".*/\1/g")
+
+  SCYLLA_RELEASE=$(echo $SCYLLA_VERSION | sed -e "s/\([[:digit:]]\+.[[:digit:]]\+\).*/\1/g")
+
+  while [ $# -gt 0 ]; do
+    case "$1" in
+      "-h" | "--help")
+        usage
+	exit 1
+        ;;
+      "--scylla-version")
+        SCYLLA_VERSION="$2"
+        shift 2
+        ;;
+      *)
+        usage
+	exit 1
+        ;;
+    esac
+  done
+
+  ARCH=$(uname -m)
+
+  if [[ "$ARCH" != "x86_64" ]]; then
+    echo "Architecture $ARCH is not supported by this installer."
+    exit 1
+  fi
+
+  # os-release may be missing in container environment by default.
+  if [ -f "/etc/os-release" ]; then
+      . /etc/os-release
+  elif [ -f "/etc/arch-release" ]; then
+      export ID=arch
+  else
+      echo "/etc/os-release missing."
+      exit 1
+  fi
+
+  if [ "$ID" = "centos" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for CentOS ..."
+    centos_install
+  elif [ "$ID" = "rhel" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for RHEL ..."
+    rhel_install
+  elif [ "$ID" = "ol" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for Oracle Linux ..."
+    oraclelinux_install
+  elif [ "$ID" = "debian" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for Debian ..."
+    debian_install
+  elif [ "$ID" = "ubuntu" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for Ubuntu ..."
+    ubuntu_install
+  else
+    echo "Operating system '$ID' is not supported by this installer."
+    exit 1
+  fi
+
+ echo "Scylla installation done!"
+}
+
+usage() {
+  cat 1>&2 <<EOF
+scylla-get
+The platform agnostic installer for Scylla.
+
+USAGE:
+    scylla-get [FLAGS] [OPTIONS]
+
+FLAGS:
+    -h, --help              Prints help information
+
+OPTIONS:
+	--scylla-version <version>                 Scylla version to install (default: $SCYLLA_VERSION)
+EOF
+}
+
+require_cmd() {
+  CMD=$1
+  if ! command -v "$CMD" &> /dev/null
+  then
+    echo "Please make sure '$CMD' is installed on this machine."
+    exit
+  fi
+}
+
+centos_install() {
+  if [[ "$VERSION_ID" = "7" || "$VERSION_ID" = "8" ]]; then
+    yum install --assumeyes --quiet epel-release
+    curl -s -L -o /etc/yum.repos.d/scylla.repo http://downloads.scylladb.com/rpm/centos/scylla-$SCYLLA_RELEASE.repo
+    yum install --assumeyes --quiet scylla
+  else
+    echo "CentOS $VERSION_ID is not supported by this installer."
+    exit 1
+  fi
+}
+
+rhel_install() {
+  if [[ "$VERSION_ID" = "8.2" ]]; then
+    yum install --assumeyes --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
Why install epel 7 on 8.2? what's special about 8.2 compared to 8.1 or 8.3?

Reply:
IIRC, I just translated the installation instructions from our web site. Will have a look.

Reply:
I reorganized the code a bit and now that "8.2" is part of a version check, which is a allow list of OS versions I tested with.